images/bind9/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM quay.io/centos/centos:stream9
+
+RUN dnf -y update && \
+    dnf -y install bind bind-utils && \
+    dnf clean all
+
+EXPOSE 53/udp 53/tcp
+
+COPY root/ / 
+
+CMD ["/usr/sbin/named", "-g", "-c", "/etc/named.conf", "-u", "named"]
```

Comment:
shouldn't this be `/etc/bind/named.conf`?

Reply:
in Debian, it's /etc/bind/named.conf
but in Centos image it's /etc/named.conf

I relied on image used on Dockerfile

Reply:
True, but I think the changes we've done would not be used as we added files (including a named.conf file) in `/etc/bind`. If you want to use the default way the Centos package provides, you should ensure the same configuration is going to be run.

Reply:
Alternatively, the current named.conf stored in root/etc/bind/ should move to root/etc.

Reply:
got it. so I will modify
CMD ["/usr/sbin/named", "-g", "-c", "/etc/named.conf",